### DISCUSSION TITLE
Interactive GUI

### DISCUSSION THREAD

**Discussion by Ms-Ajith** [chriskiehl/Gooey#841] [General]
Is it possible to make interactive gui with Gooey?
curl_parser.add_argument('add',metavar='CHOOSE A Thing ',widget ='CheckBox')
If the user clicks the Checkbox, I need a filebrowser widget to add to the existing gui. Is it possible to make it work?

Also I would like to know whether if we use optional arguments in an argument, cant we add a variable to call back?
curl_parser.add_argument("-o",'add',metavar='CHOOSE A Thing ',widget ='CheckBox')
This command is giving me error.
ValueError: invalid option string 'add': must start with a character '-'

**Comment by chriskiehl** [comment]
Please don't duplicate questions in multiple places! This was answered [here](https://github.com/chriskiehl/Gooey/discussions/839#discussioncomment-3539864)
